examples/diff.c

```diff
@@ -290,9 +305,22 @@ static void parse_opts(struct opts *o, int argc, char *argv[])
 			o->findopts.flags |= GIT_DIFF_FIND_COPIES;
 		else if (!strcmp(a, "--find-copies-harder"))
 			o->findopts.flags |= GIT_DIFF_FIND_COPIES_FROM_UNMODIFIED;
-		else if (is_prefixed(a, "-B") || is_prefixed(a, "--break-rewrites"))
-			/* TODO: parse thresholds */
+		else if (optional_str_arg(
+				break_rewrite_arg, &args, "-B", NULL) || 
+			optional_str_arg(
+				break_rewrite_arg, &args, "--break-rewrites", NULL)) {
+
 			o->findopts.flags |= GIT_DIFF_FIND_REWRITES;
+			err = extract_break_rewrites(&m, &n, *break_rewrite_arg);
+			if (err==-1) 
```

Comment:
`if (err != 0) usage("Invalid --break-rewrite argument", break_rewrite_arg);` instead of declaring `die()` ?